example-templates/placeholders/target/CACHEDIR.TAG

```diff
@@ -0,0 +1,3 @@
+Signature: 8a477f597d28d172789f06886806bc55
```

Comment:
all files under `example-templates/placeholders/target` should not be included in the PR, please remove them.